modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/Panel/index.js

```diff
@@ -22,6 +24,22 @@ const Panel = ({
 		}
 	}, [stepChecked, defaultExpanded, hasError]);
 
+	useEffect(() => {
+		var filesChanged = false;
+
+		sections?.map((section) => {
```

Comment:
This loop doesn't return nothing, so .map is not necessary for this case.

You can replace `.map` to `.forEach` or `for of`

Reply:
When we use a .map, we usually want to generate a new array, with some transformation.

If we use a .map, which does nothing, other developer can be confused.